python/ray/data/_internal/execution/operators/task_pool_map_operator.py

```diff
@@ -37,8 +38,11 @@ def __init__(
                 transform_fn, or None to use the block size. Setting the batch size is
                 important for the performance of GPU-accelerated transform functions.
                 The actual rows passed may be less if the dataset is small.
+            concurrency: The maximum number of Ray tasks to use concurrently,
+                or None to use as many tasks as possible.
             ray_remote_args: Customize the ray remote args for this op's tasks.
         """
+        self._concurrency = concurrency
```

Comment:
Nit: Don't think it really matters here, but IIRC calling `super().__init__` first is a best practice

Reply:
Updated.